src/cascadia/TerminalControl/XamlLights.cpp

```diff
@@ -49,59 +46,66 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         }
     }
 
-    // Method Description:
-    // - This function is called when there are no more target UIElements on the screen
-    // - Disposes of composition resources when no longer in use
-    // Arguments:
-    // - oldElement: unused
-    void VisualBellLight::OnDisconnected(UIElement const& /* oldElement */)
+    void CursorLight::_InitializeProperties()

```

Comment:
Is it likely that `_InitializeProperties` will ever be different between multiple lights? Or should this be in the macro too?

Reply:
It should be in the macro too, thanks for catching that